packages/devextreme/js/__internal/scheduler/appointments/m_appointment.ts

```diff
@@ -11,8 +11,8 @@ import { addNamespace } from '@js/events/utils/index';
 import dateLocalization from '@js/localization/date';
 import messageLocalization from '@js/localization/message';
 import Resizable from '@js/ui/resizable';
-import { hide, show } from '@js/ui/tooltip/ui.tooltip';
 
+import { hide, show } from '../../ui/tooltip/m_tooltip';
```

Comment:
change please to absolute path, e.g. `'@ts/ui/tooltip/m_tooltip'`

Reply:
resolved